rent_gpu/requestor/api/templates/offer-details.html

```diff
@@ -37,6 +37,136 @@ <h2>Provider status</h2>
         {% elif offer.package == 'text-webui' %}
         Use this link to connect with Text generation WebUI: <a href="http://{{request.url.hostname}}:{{ offer.port }}" target="_blank">http://{{request.url.hostname}}:{{ offer.port }}</a>
         <br/>
+
+        <style>
+            .accordion {
+              background-color: #eee;
+              color: #444;
+              cursor: pointer;
+              padding: 10px;
+              width: 100%;
+              border: none;
+              text-align: left;
+              outline: none;
+              font-size: 15px;
+              transition: 0.4s;
+            }
+
+            .active {
+              background-color: #ddd;
+            }
+
+            .accordion:hover {
+              background-color: #ccc;
+            }
+
+            .panel {
+              padding: 0 18px;
+              display: none;
+              background-color: white;
```

Comment:
You can use `bg-white` class from bootstrap instead. https://getbootstrap.com/docs/5.0/utilities/background/

Reply:
I will.

Reply:
It happened, that I can just delete it and nothing will change. I did it already.